### DISCUSSION TITLE
Gate Sequencing

### DISCUSSION THREAD

**Discussion by huiyulhy** [utiasDSL/safe-control-gym#99] [IROS 2022 Competition]
Hi! I've noticed that the gates have a sequence/order, and would like to clarify that the trajectory planned should pass through the waypoints in the same sequence instead of having the shortest path that connects them all. Thank you for clarifying!

**Comment by JacopoPan** [comment]
Hi @huiyulhy,

you are fully correct, to complete the tasks the gate should be traversed in the order specified by the YAML file.
